Comparison Engine/Comparison Engine/bin/Debug/netcoreapp3.1/barInput.json

```diff
@@ -0,0 +1,48 @@
+[
+    {
+        "barId": 1,
+        "barName": "Minusai",
+        "barLocation": "50.00000 40.000000",
+        "availableDrinks": [
+            {
+                "drinkID": 1,
+                "drinkName": "Nealkoholinis alus",
+                "price": 2.0
+            },
+            {
+                "drinkID": 2,
+                "drinkName": "Nealkoholinis vynas",
+                "price": 2.5
+            }          
+        ]
```

Comment:
Not sure if drink names are necessary in availableDrinks list but I decided to keep them for now.